tools/teal/templates/split.teal.tmpl

```diff
@@ -5,8 +5,8 @@
 // split, or single transaction, for closing the account.
 //
 // Withdrawals from this account are allowed as a group transaction which
-// sends TMPL_RCV1 exactly TMPL_RATN/TMPL_RATD times the sum of amounts
-// across both transactions.  At least TMPL_MINPAY must be sent to TMPL_RCV1.
+// sends TMPL_RCV1 and TMPL_RCV2 amounts with exactly the ratio of
+// TMPL_RATN/TMPL_RATD.  At least TMPL_MINPAY must be sent to TMPL_RCV1.
```

Comment:
Also it might be good to change the text in the `TMPL_RATD` parameter documentation line below.